src/app/extensions/quickorder/shared/quickorder-repeat-form-quantity/quickorder-repeat-form-quantity.component.ts

```diff
@@ -29,7 +29,7 @@ export class QuickorderRepeatFormQuantityComponent implements OnInit, OnChanges,
         }
       });
     this.skuControl.valueChanges.pipe(takeUntil(this.destroy$)).subscribe(control => {
```

Comment:
this looks like you could use `this.context.connect('sku', this.skuControl.valuechanges...)` instead of subscribing here.

Reply:
And the above subscribe could probably be done with an output binding, something like `this.modelChange = this.context.select().pipe(map(ctx => pick(ctx, 'quantity', 'sku')))`

Reply:
Turns out, this can be done entirely with template bindings to the formly model. Can you have a look? https://github.com/intershop/intershop-pwa/pull/844